nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -1838,7 +1838,7 @@ Divide. This is to preserve backwards compatibility and to not force rounding er
 [.function]
 === minus
 
-*Description*: [.description]#Subtracts a numeric value from the Subject.#
+*Description*: [.description]#Subtracts a numeric value from the Subject.# Does not provide handling for overflow.
```

Comment:
Shouldn't this be inside the `#` section?

Reply:
Yes it should. Nice catch